pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_import_modules.adoc

```diff
@@ -0,0 +1,19 @@
+[id="import_modules_{context}"]
+
+= Importing modules into {ProductShortName}
+
+Importing a module into {ProductShortName}.
+
+.Prerequisites
+
+* You are assigned a publisher or admin role in {ProductShortName}.
+* You are logged in.
+* You obtained the GitHub repository URL for the module you want to import.
```

Comment:
Technically it would be any accessible Git URL, not just GitHub. Our instance of Pantheon will be behind the firewall, so internal instances of GitLab would also be fine.